examples/eks/fargateprofile.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: eks.aws.crossplane.io/v1alpha1
+kind: FargateProfile
+metadata:
+  name: my-fargateprofile
+  labels:
+    examplelabelkey: "examplelabelval"
+spec:
+  forProvider:
+    region: eu-central-1
+    clusterName: my-awesome-cluster
+    podExecutionRoleRef:
+      name: fargate-pod-execution-role
```

Comment:
I'd go for the name of `IAMRole` example we have. Similarly for `clusterNameRef.name`.

Reply:
done